dask_ml/metrics/pairwise.py

```diff
@@ -165,7 +197,14 @@ def sigmoid_kernel(X, Y=None, gamma=None, coef0=1):
 }
 
 
-def pairwise_kernels(X, Y=None, metric="linear", filter_params=False, n_jobs=1, **kwds):
+def pairwise_kernels(
+    X: ArrayLike,
+    Y: Optional[ArrayLike] = None,
+    metric: str = "linear",
+    filter_params: bool = False,
+    n_jobs: int = 1,
```

Comment:
Optional, according to scikit-learn (they actually default to None. Not sure if that's a bug in dask-ml or deliberate).

Reply:
Yeah. I think I should add `Optional[int]` for now.